hack/k8s-resources/authgear.secrets.yaml

```diff
@@ -1,8 +0,0 @@
-secrets:
-  - key: db
-    data:
-      database_schema: app
-      database_url: 'postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable'
-  - key: redis
-    data:
-      redis_url: 'redis://localhost'
```

Comment:
We need to keep this file to keep app creation works in local, maybe we could rename the folder. How about change it this way?

```diff
diff --git a/.env.example b/.env.example
index 9006ecec0..43b2ec49e 100644
--- a/.env.example
+++ b/.env.example
@@ -7,11 +7,16 @@ AUTHGEAR_ENDPOINT=http://accounts.portal.localhost:3000
 DATABASE_SCHEMA=app
 DATABASE_URL=postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable
 
+ADMIN_API_HOST_TEMPLATE={{ .AppID }}.localhost:3002
+APP_HOST_SUFFIX=.localhost:3000
+
+# Enable local fs config source
 CONFIG_SOURCE_TYPE=local_fs
 CONFIG_SOURCE_DIRECTORY=./var
 
-ADMIN_API_HOST_TEMPLATE={{ .AppID }}.localhost:3002
-APP_HOST_SUFFIX=.localhost:3000
+# Enable database config source
+#CONFIG_SOURCE_TYPE=database
+#CUSTOM_RESOURCE_DIRECTORY=./hack/custom-resources
 
 # This is for portal to create k8s ingress when creating domains
 #DOMAIN_IMPLEMENTATION=kubernetes
diff --git a/README.md b/README.md
index a3ee7df2b..2c16ec46f 100644
--- a/README.md
+++ b/README.md
@@ -161,7 +161,7 @@ To setup multi-tenant mode:
    ```
 
 4. Enable multi-tenant mode in Authgear & portal server:
-   refer to `.env.example.k8s` for example environment variables to set
+   refer to `.env.example` for example environment variables to set
 
 ## Portal
 
diff --git a/hack/custom-resources/authgear.secrets.yaml b/hack/custom-resources/authgear.secrets.yaml
new file mode 100644
index 000000000..ea0fb64e5
--- /dev/null
+++ b/hack/custom-resources/authgear.secrets.yaml
@@ -0,0 +1,8 @@
+secrets:
+  - key: db
+    data:
+      database_schema: app
+      database_url: 'postgres://postgres:postgres@localhost:5432/postgres?sslmode=disable'
+  - key: redis
+    data:
+      redis_url: 'redis://localhost'
```

Reply:
Let me add this commit and merge it